lib/index.js

```diff
@@ -45,8 +47,8 @@ class Linter {
     }
 
     let rules = this.config.loadedRules;
+    let astPlugins = [TransformDotComponentInvocation];
```

Comment:
@rwjblue I am not understanding, how the linter errors are shown for the first time when i run `ember s`. Is it using some cached output? It seems like that.